apps/cookbook/src/app/showcase/accordion-showcase/accordion-showcase.component.html

```diff
@@ -34,18 +34,22 @@ <h2>Accordion inside a card</h2>
     <cookbook-accordion-card-example #accordionCardExample></cookbook-accordion-card-example>
   </cookbook-example-viewer>
 
-  <h2>Accordion inside card with items</h2>
+  <h2>Accordion with list</h2>
   <p>
-    Since kirby-items come with their own built-in padding the accordions below have
-    <code>hasPadding</code>
-    set to
-    <code>false</code>
-    to avoid double padding. This allows for content inside items to be aligned with the accordion header.
+    Placing a list within an accordion item will automatically set the correct styling such that the
+    list looks correct without having to adjust the components.
   </p>
-  <cookbook-example-viewer [html]="accordionWithListItemsExample.template">
-    <cookbook-accordion-with-items-example
-      #accordionWithListItemsExample
-    ></cookbook-accordion-with-items-example>
+  <cookbook-example-viewer [html]="accordionWithListExample.template">
+    <cookbook-accordion-with-list-example
+      #accordionWithListExample
+    ></cookbook-accordion-with-list-example>
+  </cookbook-example-viewer>
+
+  <h2>Accordion with list inside a card</h2>
```

Comment:
Consider something like this, to couple it a bit better with the 'Accordion with list' example - note the h3 here, to make it a subheading instead.
```suggestion
  <h3>Additional styling</h3>
  <p>Similarly to the simple accordion, the accordion with a list can also be wrapped in a Kirby Card to make it stand out from the canvas.</p>
```

Consider just